docs/source/credentials.rst

```diff
@@ -0,0 +1,37 @@
+Credentials
+***********
+
+Integration with Git
+====================
+
+Git and DataLad can use each other's credential system.
+In order to allow Git to query credentials from DataLad, the git credential helper delivered with DataLad needs to be configured.
+That is, a section like this needs to be part of one's git config file::
+
+  [credential "http://*.ds.inm7.de"]
+    helper = "datalad"
+  [credential "https://*.ds.inm7.de"]
+    helper = "datalad"
```

Comment:
This makes the impression there is always a need to have two config section pairs, but this just for this particular case

```suggestion
  [credential "https://*.data.example.com"]
    helper = "datalad"
```